editor/connections_dialog.cpp

```diff
@@ -446,7 +446,7 @@ ConnectDialog::ConnectDialog() {
 	type_list->add_item("Quaternion", Variant::QUATERNION);
 	type_list->add_item("AABB", Variant::AABB);
 	type_list->add_item("Basis", Variant::BASIS);
-	type_list->add_item("Transform", Variant::TRANSFORM3D);
+	type_list->add_item("Transform3D", Variant::TRANSFORM3D);
```

Comment:
For the reference, this dialog doesn't include a number of Variant types: Transform2D, Vector2i, Vector3i, Rect2i, and everything after NodePath.
I don't know if this is intentional and/or if this should be fixed.